doc/tutorials/gaussian_transformation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>Here and elsewhere (and since I just corrected that mistake in my PR :):</p><p><br></p><p>The quotation marks ``..`` (or even `..`?) as in <code>Continuous Variable</code> turn things into verbatim font which is reserved for code and values/functions used in the code. I think you want to use *..* for italics? Or does it just render things weird in the reviewnb editor?</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I thought it highlighted text in the jupyter notebook. You are right. When rendered into its html version, it appears in the verbatim font. I will change this in all notebooks where necessary. Thanks  </p>